lightly/api/bitmask.py

```diff
@@ -0,0 +1,197 @@
+""" Download from Lightly API """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+from typing import List
+
+
+def _hex_to_int(hexstring: str) -> int:
+    """Converts a hex string representation of an integer to an integer.
+    """
+    return int(hexstring, 16)
+
+
+def _bin_to_int(binstring: str) -> int:
+    """Converts a binary string representation of an integer to an integer.
+    """
+    return int(binstring, 2)
+
+
+def _int_to_hex(x: int) -> str:
+    """Converts an integer to a hex string representation.
+    """
+    return hex(x)
+
+
+def _int_to_bin(x: int) -> str:
+    """Converts an integer to a binary string representation.
+    """
+    return bin(x)
+
+
+def _get_nonzero_bits(x: int) -> List[int]:
+    """Returns a list of indices of nonzero bits in x.
+    """
+    offset = 0
+    nonzero_bit_indices = []
+    while x > 0:
+        # if the number is odd, there is a nonzero bit at offset
+        if x % 2 > 0:
+            nonzero_bit_indices.append(offset)
+        # increment the offset and divide the number x by two (rounding down)
+        offset += 1
+        x = x // 2
+    return nonzero_bit_indices
+
+
+def _invert(x: int) -> int:
```

Comment:
Let's create an issue to implement this, should be a quick one.

Reply:
See https://github.com/lightly-ai/lightly/pull/131/commits/48dfca68d534e09dc09f0c4f01e0b9b4cbd32bac